### PR TITLE
Update to responsive design

### PR DESCRIPTION
**Pull request by msDestiny14**
https://github.com/BabylonJS/ThePirateCove/issues/73

### CODE REVIEW

File: guiEditor/src/components/propertyTab/propertyTabComponent.tsx [hunk 1]
```diff
@@ -446,19 +453,19 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
                         noDirectUpdate={true}
                         onSelect={(value: any) => {
                             this._sizeOption = value;
-                            if (this._sizeOption != (sizeOptions.length - 1)) {
+                            if (this._sizeOption != (sizeOptions.length)) {
```
Comment (RaananW) [inline guiEditor/src/components/propertyTab/propertyTabComponent.tsx:456]:
```suggestion
                            if (this._sizeOption !== (sizeOptions.length)) {
```

Always use `===` or `!==`, unless it is really needed to compare different types. It's less of a performance issue and more of a runtime-error-handling thing. this should never work if one of those types is different than the other.

File: guiEditor/src/components/propertyTab/propertyTabComponent.tsx [hunk 2]
```diff
@@ -446,19 +453,19 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
...
-                    />
-                    {this._sizeOption == (sizeOptions.length - 1) &&
+                    />}
+                    {this._sizeOption == (sizeOptions.length) &&
                         <div className="divider">
                             <FloatLineComponent
                                 icon={canvasSizeIcon}
                                 iconLabel="Canvas Size"
-                                label="X"
+                                label="W"
```
Comment (RaananW) [inline guiEditor/src/components/propertyTab/propertyTabComponent.tsx:468]: was it intentional to change the label here ( i assume this is the keycode used?)

**Review by RaananW — COMMENTED** [review]
small nits.

File: guiEditor/src/components/propertyTab/propertyTabComponent.tsx (hunk 2 above)
Comment (msDestiny14) [reply guiEditor/src/components/propertyTab/propertyTabComponent.tsx:468]: Yes. This is to match the design. W for width and H for height.

**Pull request merged**